packages/addons/service/multimedia/vdr-addon/source/bin/vdr.start

```diff
@@ -133,9 +133,24 @@ if [ "$ENABLE_XMLTV2VDR" == "true" ] ; then
     cp $source /var/lib/epgsources
   done
 fi
+
 if [ "$ENABLE_RESTFULAPI" == "true" ] ; then
-  VDR_ARG="$VDR_ARG -P 'restfulapi --ip $RESTFULAPI_IP --port $RESTFULAPI_PORT'"
+  VDR_ARG="$VDR_ARG -P 'restfulapi --ip $RESTFULAPI_IP --port $RESTFULAPI_PORT"
```

Comment:
appending args this way is a bit confusing. better use VDR_RESTFUL_ARG and append it later below.